src/System.Management.Automation/engine/Utils.cs

```diff
@@ -8,6 +8,7 @@
 using System.Management.Automation.Internal;
 using System.Management.Automation.Security;
 using System.Reflection;
+using Microsoft.PowerShell;
```

Comment:
Is this actually needed? The only other change to this file is the removal of the GetEncoding logic.